sbt_wrapper/Dockerfile

```diff
@@ -37,4 +39,4 @@ RUN /usr/local/sbt/bin/sbt sbtVersion
 # According to https://stackoverflow.com/questions/46475649/sbt-hangs-during-tests-on-travis-ci,
 # this might be due to sbt not having enough memory and swapping too much.
 # So increase memory and see what happens
-ENTRYPOINT ["/usr/local/sbt/bin/sbt", "-J-Xss6M", "-J-Xms2G", "-J-Xmx3G"]
\ No newline at end of file
+ENTRYPOINT ["/usr/local/sbt/bin/sbt", "-J-Xss6M", "-J-Xms4G", "-J-Xmx4G", "-J-XX:MaxMetaspaceSize=2G"]
```

Comment:
Worth updating the comment, possibly linking to our Slack discussion so people know how we arrived at these values? Certainly the Travis comment is about to not apply.